ome/omero-web
# Chown webclient

Adds ```chown``` functionality to webclient.

![Screenshot 2020-03-12 at 11 23 57](https://user-images.githubusercontent.com/900055/76516807-06a92400-6454-11ea-945b-266520e160e6.png)

To test:
 - Log-in as a Group Owner or Admin that has 'Chown' privilege (other users should have chown option disabled and tooltip to explain why)
 - Choose P/D/I or S/P and use right-click menu "Change Owner..."
 - Dialog should list other members of the group (not the owner of the data if all the selected objects belong to the same user)
 - Also, dry-run will run immediately (as long as there is at least 1 other member of the group)
 - Dry-run has a cancel button, which attempts to cancel - but not sure this really prevents much work on the server.
 - Dry-run results should show what won't be chowned.
 - Choose new Owner and click OK.
 - Activities dialog will show when Done.
 - The selected object(s) and any child objects should all now belong to the new owner

NB: Some links can be lost when chown.
E.g. As a group Owner, I put MY Image into another user's Dataset. Then I Change owner of the Dataset to ME. The Image will get removed from the Dataset (because it didn't belong to the same owner as the Dataset?).
